bower.json

```diff
@@ -12,5 +12,8 @@
     "adapt-plugin",
     "adapt-component"
   ],
+  "pluginDependencies": {
+    "adapt-contrib-mcq": "*"
```

Comment:
I think this might need to be ">=2.1" (as the current GMCQ is dependent on the Model/View version of MCQ)